source/CIS/cis-0.rst

```diff
@@ -0,0 +1,117 @@
+.. _CIS-0:
+
+=========================
+CIS-0: Standard Detection
+=========================
+
+.. list-table::
+   :stub-columns: 1
+
+   * - Created
+     - Jul 14, 2022
+   * - Draft version
+     - 1
+   * - Supported versions
+     - | Smart contract version 1 or newer
+       | (Protocol version 4 or newer)
+   * - Standard identifier
+     - ``CIS-0``
+
+.. warning::
+
+   This standard is still a draft and significant changes might still be made.
+
+Abstract
+========
+
+A standardized interface for how a smart contract publish supported smart contract standards.
```

Comment:
It is rather strange to state it as "publish" since no publishing is being done.

Perhaps "report" or "indicate"?